app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
       projectName = projectName || currentProject;
       var rules = getRulesForProject(projectName);
       if (permissiveMode) {
         return true;
       } else if (rules) {
         if (rules[resource]) {
-          return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+          return checkRulesForResourceVerb(rules, resource, verb);
         } else if (rules['*']) {
-          return _.contains(rules['*'], verb) || _.contains(rules['*'], '*');
+          return checkRulesForResourceVerb(rules, '*', verb);
+        } else if (rules['*/*']) {
```

Comment:
it looks like `normalizeRules` *sometimes* includes apiGroup with a slash, which makes it impossible to distinguish between {apiGroup:"builds",resource:"logs"} and {apiGroup:"",resource:"builds/logs"}